Cesium3DTilesSelection/src/RasterOverlayTileProvider.cpp

```diff
@@ -565,15 +217,15 @@ struct LoadResult {
  * @return The `LoadResult`
  */
 static LoadResult createLoadResultFromLoadedImage(
-    const CesiumGeometry::QuadtreeTileID& tileId,
     const std::shared_ptr<IPrepareRendererResources>& pPrepareRendererResources,
     const std::shared_ptr<spdlog::logger>& pLogger,
     LoadedRasterOverlayImage&& loadedImage) {
   if (!loadedImage.image.has_value()) {
     SPDLOG_LOGGER_ERROR(
         pLogger,
         "Failed to load image for tile {}:\n- {}",
-        Cesium3DTilesSelection::TileIdUtilities::createTileIdString(tileId),
+        "TODO",
```

Comment:
There are currently 12 appearances of `TODO` in the new file state. Some of them might already have been in the old state, but this one ^ should be fixed, and some other ones should be moved into issues.